components/browser/state/src/main/java/mozilla/components/browser/state/state/TabSessionState.kt

```diff
@@ -40,6 +41,7 @@ data class TabSessionState(
     override val source: SessionState.Source = SessionState.Source.NONE,
     val parentId: String? = null,
     val lastAccess: Long = 0L,
+    val createdAt: Long = 0L,
```

Comment:
I wonder if it makes sense to default this to `System.currentTimeMillis()` here as well. I'm fine with the current version too, but maybe Christian feels otherwise.

Reply:
Yes, I agree, it would make sense to initialize with `now`. The tab can be created without being accessed to it makes sense for `lastAccess` to be `0` initially, but `createdAt` be set to `now`.

Reply:
`now` as in writing something like:

`fun now() = System.currentTimeMillis()` ?
All other `now`'s in AC seem to measure other things, and the ones from android require API 26.

Reply:
@codrut-topliceanu ah no, just as @jonalmeida mentioned as the initial value `val createdAt = System.currentTimeMillis()`, instead of initializing with `0`.